### ISSUE TITLE
Extensions does not appear in "Add Step" dialog

### ISSUE THREAD

**Issue by JPGRodrigues** [rfennell/vNextBuild#16]
I purchased this extension but it does not appears in the "Add Step" dialog!

**Comment by rfennell** [comment]
I assume you are using the Generate Release notes extension, if so this is a duplicate of the issue #11 

Some users are experiencing this issue, but others are not. I have a support call open with Microsoft to try to get to the bottom of this.

If you could email me (richard@blackmarble.co.uk) your VSTS instance name I can pass it onto Microsoft to assist in their daignosis

**Comment by rfennell** [comment]
After lengthy discussion with MS, the issues with the lack of tasks appearing seems to be related some unfortunate timing of my uploads and some updates.

The solution appears to be to uninstall my extension, pause, then re add it to your VSTS instance. The tasks should appear instantly, however I have seen that they may take a few minutes to appears (some cache issue)

Let me know if you are now unblocked

**Comment by rfennell** [comment]
Closing call as heard nothing more
